.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "interactive"
```

Comment:
This is odd, we don't generally check these things into github...

Reply:
Can you remove the .json folder, and add it to .gitignore please ?

The idea is that people can use any IDE they want, so having vscode-specific stuff in there isn't necessary I think.